rust-toolchain.toml

```diff
@@ -0,0 +1,2 @@
+[toolchain]
+channel="1.60.0"
```

Comment:
It looks now more complicated.
My understanding is that there are two use cases:
* developers building repo
* CI building repo
For the first one, we just want to stick with 1.60 and that's all.
For CI, we need to specify version for both stable and nightly. The problem is that it is impossible using plain rust tools.

It looks like in other projects people solve the problem on CI level by, for example, using github workflows actions [tokio CI file](https://github.com/tokio-rs/tokio/blob/master/.github/workflows/ci.yml#L13)

So although the proposed solution with two cargo files looks like working, I feel it might  be more complicated than the current approach for CI at least.

Reply:
Developers need to use stable for building and testing, and nightly for fmt and clippy, so both CI and developer workflows need easy access to both.

A lot of people aren't aware of needing to use the `./cargo` script, so with this at least, we don't need to use a separate script for building and testing, which I would count as a win.

As you say, since it's impossible to have two toolchains with the normal tools, the separate `cargo-nightly` script still allows us to easily run nightly.  I'm totally open to other solutions though!

Reply:
Btw I ended up doing this for SPL -- the monorepo is more complicated, but I think it can use the same pattern: https://github.com/solana-labs/solana-program-library/pull/3466

Reply:
Thanks for pointing to this PR, will try the same approach